src/ast_utils.h

```diff
@@ -187,7 +189,14 @@ struct EffectAnalyzer : public PostWalker<EffectAnalyzer> {
   }
 
   void visitCall(Call *curr) { calls = true; }
-  void visitCallImport(CallImport *curr) { calls = true; }
+  void visitCallImport(CallImport *curr) {
+    calls = true;
+    if (debugInfo) {
+      // debugInfo call imports must be preserved very strongly, do not
```

Comment:
yeah, we could. call imports are rare enough that i doubt it matters, and this is debug info mode after all. what do you think?